package.json

```diff
@@ -103,14 +102,14 @@
     "commitplease": "3.2.0",
     "concurrently": "8.2.2",
     "core-js-bundle": "3.36.0",
-    "eslint": "8.56.0",
+    "eslint": "8.57.0",
     "eslint-config-jquery": "3.0.2",
     "eslint-plugin-import": "2.29.1",
     "exit-hook": "4.0.0",
-    "express": "4.18.2",
+    "express": "4.18.3",
     "express-body-parser-error-handler": "1.0.7",
     "globals": "14.0.0",
-    "husky": "8.0.3",
+    "husky": "9.0.11",
```

Comment:
On the 3.x PR, you also changed `husky install` to `husky`. Isn't that required here as well?

Reply:
Thanks, I meant to do that. Both technically work, but the first is deprecated.